src/standalone/Vegetation/photosynthesis.jl

```diff
@@ -1,4 +1,33 @@
-export FarquharParameters, FarquharModel, C3, C4
+export SIFParameters, FarquharParameters, FarquharModel, C3, C4
+
+abstract type AbstractPhotosynthesisModel{FT} <: AbstractCanopyComponent{FT} end
+
+"""
+    SIFParameters{FT<:AbstractFloat}
+
+The required parameters for the SIF parameterisation [give citation].
+$(DocStringExtensions.FIELDS)
+"""
+@kwdef struct SIFParameters{FT <: AbstractFloat}
+    ""
+    kf::FT = FT(0.05)
+    ""
+    kd_p1::FT = FT(0.03)
+    ""
+    kd_p2::FT = FT(0.0273)
+    ""
+    min_kd::FT = FT(0.087)
+    ""
+    kn_p1::FT = FT(6.2473)
+    ""
+    kn_p2::FT = FT(0.5944)
+    ""
+    kp::FT = FT(4.0)
```

Comment:
Rate coefficient for photochemical quenching - can ask Renato where he got it

Reply:
Kp0 is the maximum photochemistry, eq 5b in:

Li, R., Lombardozzi, D., Shi, M., Frankenberg, C., Parazoo, N. C., Köhler, P., Yi, K., Guan, K., & Yang, X. (2022). Representation of Leaf-to-Canopy Radiative Transfer Processes Improves Simulation of Far-Red Solar-Induced Chlorophyll Fluorescence in the Community Land Model Version 5. Journal of Advances in Modeling Earth Systems, 14(3), e2021MS002747. https://doi.org/10.1029/2021MS002747